sdk/src/cose_sign.rs

```diff
@@ -204,7 +204,12 @@ fn build_headers(signer: &dyn Signer, data: &[u8], alg: SigningAlg) -> Result<(H
     };
 
     let certs = signer.certs()?;
-    let ocsp_val = signer.ocsp_val();
+
+    let ocsp_val = if _sync {
```

Comment:
This makes me think I should (someday) update async_generic to trim away the `.await` clause in the synchronous case. (Not going to happen soon, sadly.)